django_extensions/management/commands/print_settings.py

```diff
@@ -111,3 +111,13 @@ def include_attr(attr, settings):
     def print_simple(a_dict):
         for key, value in a_dict.items():
             print("%-40s = %r" % (key, value))
+
+    @staticmethod
+    def safe_settings():
+        try:
+            return SafeExceptionReporterFilter().get_safe_settings()
```

Comment:
If django < 3.0 does not have this class then the import above will already have failed and this code is meaningless.

Either the comment below is wrong or we should catch `ImportError` during importing.

FYI: This entire code can be replaced to function at startup time which seem to be preferred to this way if it served no additional function.